addons/hellfire/CfgMagazineWells.hpp

```diff
@@ -0,0 +1,4 @@
+class CfgMagazineWells {
+    class GVAR(magwell_K) {};
```

Comment:
It's in a magwell config and alway use in magwell related entries. CBA also doesn't prepend "magwell". Just "K" or "N" will do ("ace_hellfire_k") unless that's already taken (does that even matter in this case)?

Reply:
`ace_hellfire_K` should be fine

Reply:
>unless that's already taken (does that even matter in this case)

No. Doesn't matter if already taken.